From 0d7d01ed2e919007edffb756fab62dd431095389 Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Sun, 27 Nov 2011 16:43:11 +0000
Subject: Disable failing GNU/Hurd tests io/pipe.t

These tests currently fail on GNU/Hurd, so disable them

Bug-Debian: http://bugs.debian.org/650187

Patch-Name: debian/hurd_test_skip_pipe.diff
---
 t/io/pipe.t |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/t/io/pipe.t b/t/io/pipe.t
index 9c165a3..d69f1c4 100644
--- a/t/io/pipe.t
+++ b/t/io/pipe.t
@@ -9,6 +9,9 @@ BEGIN {
     if (!$Config{'d_fork'}) {
         skip_all("fork required to pipe");
     }
+    elsif ($^O eq 'gnu') {
+        skip_all('fails on GNU/Hurd (Debian #650187)');
+    }
     else {
         plan(tests => 24);
     }
